docker/lr-pb/Makefile

```diff
@@ -2,11 +2,14 @@ VERSION = 0.1.36
 TAG1 = us.gcr.io/broad-dsp-lrma/lr-pb:$(VERSION)
```

Comment:
again, with these major changes, it deserves a version change.

Reply:
Sounds good.  I bumped it to `0.1.37`